resolvers/index.js

```diff
@@ -19,11 +19,13 @@ const queryResolvers = {
    * Retrive user using Cookies
    */
   currentUser: async (root, args, context) => {
-    if (!context.User.external_id) {
+    const { User: user } = context
+
+    if (!user || !user.external_id) {
```

Comment:
**nice to have.** probably we should add `optional-chaining`
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
- https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining

Reply:
I like this, let me see if I can integrate that into this PR